### PR TITLE
Update of JMS Livelabs Lab 6 Install Management Agent on your Managed Instances 

### PR DESCRIPTION
**Pull request by KianMun-oracle**
# Add here your WMS ID

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

File: java-management/set-up-of-management-agent/set-up-of-management-agent.md [hunk 1]
```diff
@@ -56,86 +56,91 @@ This task is for the installation of Management Agent in an OCI Host(OCI Compute
...
+1. To Download agent installer software, in the Oracle Cloud Console, open the navigation menu and click **Observability & Management**, and then click **Fleets** under **Java Management**.
 
 ![image of navigate to jms-console](images/console-navigation-jms.png)
 
 2. Select the Fleet created in [Lab 3](?lab=setup-a-fleet).
-3. Click **configure managed instances**
+3. Click **Download Agent Installer**
 
-![image of clicking configuration](images/click-configuration.png)
+![image of clicking configuration](images/DownloadAgentInstaller.png)
```
Comment (sktruong-oracle) [inline java-management/set-up-of-management-agent/set-up-of-management-agent.md:68]: follow the same naming covention (dash-case). Applies to other file names as well

**Review by sktruong-oracle — APPROVED** [review]

File: java-management/set-up-of-management-agent/set-up-of-management-agent.md (hunk 1 above)
Comment (KianMun-oracle) [reply java-management/set-up-of-management-agent/set-up-of-management-agent.md:68]: Changed. Affected files: Agent-installer-close.png, Download-Agent-Installer.png, Select-Agent-Installer.png

Comment (sktruong-oracle) [reply java-management/set-up-of-management-agent/set-up-of-management-agent.md]: can you make it all lower case also?

Comment (KianMun-oracle) [reply java-management/set-up-of-management-agent/set-up-of-management-agent.md]: All affected files changed to lower case.

Comment (sktruong-oracle) [reply java-management/set-up-of-management-agent/set-up-of-management-agent.md]: image
some file names are still ProperCase

Comment (KianMun-oracle) [reply java-management/set-up-of-management-agent/set-up-of-management-agent.md]: My bad. All affected files changed.

**Pull request merged**